en/docs/insights/choreo-insights-api.md

```diff
@@ -1162,6 +1162,207 @@ Time filter
 
 Successful API usage by application filter.
 
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getSuccessAPIsUsageByApplicationsWithOnPremKey</strong></td>
+<td valign="top">[<a href="#successapiusagebyapplication">SuccessAPIUsageByApplication</a>]</td>
+<td>
+
+Returns a list of successful hit counts (i.e., 2xx response codes) within the specified time range for each API,
+grouped by each application. Optionally, you can configure the <code>successAPIUsageByAppFilter</code> parameter to further
+filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">onPremKey</td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Value of the On-Prem key
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">successAPIUsageByAppFilter</td>
+<td valign="top"><a href="#successapiusagebyappfilter">SuccessAPIUsageByAppFilter</a></td>
+<td>
+
+Successful API usage by application filter.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>getCustomReportOvertime</strong></td>
+<td valign="top"><a href="#customreportovertime">CustomReportOvertime</a></td>
+<td>
+
+Returns summary of hits/ latency data for the inquired time frame grouped into timeSpans
```

Comment:
```suggestion
Returns the summary of hits or latency data as specified for the selected time duration, grouped by time spans.
```
- What did you mean by "timeSpans" here and was there a reason to have it as one word and in camel case?
- Does it return either the summary of hits or latency as specified by the user? Is so, we should mention that.
- What is the difference between the "time frame" and "time span" here? Are time spans standard time granularities?

Reply:
- Time spans are standard time granularities. The relevant parameter is declared in camel case in the schema. Other than that there is no need to have it as a single word here in camel case.
- Yes, it returns either the summary of hits or latency depending on the metric specified by the user. 
- Time frame is the 'from x to y' time period that the user specified in the 'timeFilter' and time spans are standard time granularities as mentioned before.